code/modules/mob/living/emote.dm

```diff
@@ -286,15 +292,10 @@
 /datum/emote/living/laugh/can_run_emote(mob/living/user, status_check = TRUE , intentional)
 	return ..() && user.can_speak(allow_mimes = TRUE)
 
-/datum/emote/living/laugh/get_sound(mob/living/user)
+/datum/emote/living/laugh/get_sound(mob/living/carbon/user)
 	if(!ishuman(user))
```

Comment:
After this check you can assign ``var/mob/living/carbon/human/human_user = user``